pkg/awsclicompat/awsclicompat.go

```diff
@@ -39,5 +40,10 @@ func NewSession(region, profile string) *session.Session {
 
 	sess := session.Must(session.NewSessionWithOptions(opts))
 
+	if roleARN != "" {
+		creds := stscreds.NewCredentialsWithClient(sts.New(sess), roleARN)
+		return sess.Copy(&aws.Config{Credentials: creds})
+	}
```

Comment:
Here is the main feature. When passing `roleARN`, we can call aws api using sts.